src/ec/packages/ec-component-timeline/ecl-timeline.html.twig

```diff
@@ -69,7 +69,7 @@ Parameters:
     {% for _item in _items %}
       {# Decide on per item if it should display. #}
       {% set extra_class = (loop.index > _display_amount and loop.revindex > _bottom_amount) ? ' ecl-timeline2__item--collapsed' : '' %}
-      <li class="ecl-timeline2__item{{ extra_class }}" id="{{ _item.id }}">
+      <li class="ecl-timeline2__item{{ extra_class }}" {% if item.id is not empty %} id="{{ _item.id }}" {% endif %}>
         <div class="ecl-timeline2__label">{{ _item.label }}</div>
         {% if _item.title is not empty %}
```

Comment:
Also this is not anything you touched, but i don't see the reason why we are assuming that _item.title is going to be in place, twig is not php, !empty doesn't mean "if isset and is not empty", we need to check also for the property itself, if `_item.title is set and _item.title is not empty`